vtex/faststore
# Feat: Next 13 - Global Section - `Toast` Section - SFS-1021

## What's the purpose of this pull request?

This PR aims to migrate the `Toast`  from the pages router to the app router (next 13). 


## How to test it?

- Run the local server (`yarn dev` in core package);
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with thew `Toast` section on the top of the page and without errors.
- You can add a wrong zip code and a lot of products to make sure the `Toast` will appear.

<img width="1019" alt="Screenshot 2024-06-13 at 18 04 58" src="https://github.com/vtex/faststore/assets/11325562/b378527f-4dc5-4619-a9b6-f2e2ac5192f0">
<img width="1036" alt="Screenshot 2024-06-13 at 18 05 14" src="https://github.com/vtex/faststore/assets/11325562/ba3833a7-fc66-4df5-b69b-59b3c9bec0b4">
